### ISSUE TITLE
Refactor to ES6 syntax and standards

### ISSUE THREAD

**Issue by TimMikeladze** [Tacoma-JS/weatherapp#9]
The Javascript can be updated to use ES6 syntax and modules.

**Comment by NorthDecoder** [comment]
Agreed @TimMikeladze . And here are some references to help us with that:
* [es6-features](http://es6-features.org/#Constants)
* [compat-table](https://kangax.github.io/compat-table/es6/)
* [caniuse](http://caniuse.com/#home)
* [practical-modern-javascript](https://github.com/mjavascript/practical-modern-javascript) a book
* TL;DR 😸  [ECMA-262](http://www.ecma-international.org/ecma-262/6.0/index.html) 6th edition
